tfhe/src/core_crypto/algorithms/lwe_programmable_bootstrapping/fft64.rs

```diff
@@ -1071,3 +1071,77 @@ pub fn programmable_bootstrap_lwe_ciphertext_mem_optimized_requirement<OutputSca
 ) -> Result<StackReq, SizeOverflow> {
     bootstrap_scratch::<OutputScalar>(glwe_size, polynomial_size, fft)
 }
+
+/// Memory optimized version of batch_programmable_bootstrap_lwe_ciphertext, the caller must
```

Comment:
we don't have the function mentioned in here, so a small description of what it does ( loading a key bit and doing the computation for all ciphertexts and then moving onto the next key bit for example)